### DISCUSSION TITLE
Transfer Passkeys clean install

### DISCUSSION THREAD

**Discussion by growbug** [keepassxreboot/keepassxc#11264] [Q&A] [answered]
Hello all,

Recent user of KeepassXC after using Keepass for many years, OS is fedora 40 with Gnome 46 desktop environment, KeepassXC v 2.7.8 Appimage.

One or two questions please on what's possible when transferring passkeys, firstly using Appimage is the functionality the same as with KeepassXC PPA or Flatpak?

Is it possible to store passkeys as you would passwords, then simply copy the kbx file to a new or clean install without any other additional steps, my understanding is passkeys are device specific tied to the TPM module on Windows implementation but unsure on Linux?

Reading the KeepassXC guide is helpful, as are YT videos on the subject but still a tad sceptical - thanks for any pointers.

**Comment by droidmonkey** [comment, marked as answer]
All passkey information is stored in the kdbx file, as long as you registered the passkey with keepassxc. Moving your passkeys is as simple as moving the kdbx file. We do recommend keeping backups and version control (eg, cloud storage, sync thing, etc).

I recommend flatpak over appimage. The appimage has some weird quirks because that packaging format isn't a pure isolation from your system.

**Reply by growbug** [reply]
Thanks for the speedy reply, 

Maybe I have misunderstood the concept but in order for the passkey (private key) to work it needs to be validated by the device/credential manager that generated it? 

In summary, is the passkey not unique to one machine ?

I'll swap to flatpak on my next install, thanks

**Reply by droidmonkey** [reply]
KeePassXC serves as the credential validator

**Reply by growbug** [reply]
ahhh with you, that solves that one, cheers
